widgetssdk/src/main/java/com/glia/widgets/GliaWidgetsConfig.kt

```diff
@@ -64,12 +61,25 @@ class GliaWidgetsConfig private constructor(builder: Builder) {
         suppressPushNotificationsPermissionRequestDuringAuthentication = builder.suppressPushNotificationsPermissionRequestDuringAuthentication
     }
 
+    private fun requireRegion(region: Region?, regionString: String?): Region {
+        requireGlia(region == null || regionString == null) { "`setRegion(region: Region)` and `setRegion(region: String)` are mutually exclusive" }
+
+        return when {
+            region != null -> region
```

Comment:
A suggestion. You've explained this line to me. But if you haven't - it would be a bit more dificult to understand this logic `region != null -> region` So it might be good to add a comment that it's for backwards compatibility, to support an old approach (if I understood correctly). 🙂

Reply:
it should be clear after [these](https://github.com/salemove/android-sdk-widgets/pull/1423/commits/ea47077405c3bdfaea29d6f82c58b93bc4efce1a) changes